ui/client/components/modals/Dialogs.js

```diff
@@ -1,37 +1,42 @@
-import React from "react";
+import React from "react"
+import CalculateCountsDialog from "./CalculateCountsDialog"
+import CompareVersionsDialog from "./CompareVersionsDialog"
 import ConfirmDialog from "./ConfirmDialog"
+import GenerateTestDataDialog from "./GenerateTestDataDialog"
+import InfoModal from "./InfoModal"
 import ProcessActionDialog from "./ProcessActionDialog"
 import SaveProcessDialog from "./SaveProcessDialog"
-import GenerateTestDataDialog from "./GenerateTestDataDialog"
-import CalculateCountsDialog from "./CalculateCountsDialog"
-import CompareVersionsDialog from "./CompareVersionsDialog"
-import InfoModal from "./InfoModal";
-
 
-class AllDialogs extends React.Component{
-    render() {
-        return (
-            <div>
-                <ConfirmDialog/>
-                <ProcessActionDialog />
-                <InfoModal/>
-                <SaveProcessDialog/>
-                <GenerateTestDataDialog/>
-                <CalculateCountsDialog/>
-                <CompareVersionsDialog/>
-            </div>
-        );
-    }
+class AllDialogs extends React.Component {
+  render() {
+    return (
+        <div>
+          <ConfirmDialog />
```

Comment:
We do not have consistent style of files formatting - should we add spaces before closing tags?

Reply:
fixed and added lint setting